apps/app/src/app/(main-layout)/authors/page.tsx

```diff
@@ -0,0 +1,18 @@
+import { Author } from "../../../components/Author/Author";
+import { StaticPageContainer } from "../../../components/StaticPageContainer";
+import { getAllContributors } from "../../../lib/github";
+
+export default async function AuthorsPage() {
+	const contributors = await getAllContributors();
+
+	return (
+		<StaticPageContainer>
+			<h1 className="mb-8 text-4xl font-bold">Autorzy</h1>
```

Comment:
Czemu na pozostałych stronach tytuł strony jest h2, a tu h1?
W h1 będzie raczej zawsze DevFAQ w headerze strony (albo powinien być)
```suggestion
			<h2 className="mb-8 text-4xl font-bold">Autorzy</h2>
```